query-engine/core/src/schema_builder/input_types/fields/data_input_mapper/create.rs

```diff
@@ -126,7 +126,7 @@ impl DataInputFieldMapper for CreateDataInputFieldMapper {
         // If the composite field in _not_ on a model, then it's nested and we're skipping the create envelope for now.
         // (This allows us to simplify the parsing code for now.)
         let mut input_types = if cf.container().as_model().is_some() {
-            vec![shorthand_type.clone(), envelope_type]
+            vec![envelope_type, shorthand_type.clone()]
```

Comment:
Any particular reason for the ordering?

Reply:
For the playground to pick up the envelope types instead of the shorthand ones. Makes it a lot easier to navigate the API.